_posts/2020-09-07-introduction-to-system-tests.md

```diff
@@ -0,0 +1,281 @@
+Everything is deployed into a real Kubernetes cluster.
+All test cases with related test suites can be found in the [systemtest](https://github.com/strimzi/strimzi-kafka-operator/tree/master/systemtest) 
+module.
+Also worth mentioning is that our system tests currently take around 30 hours. 
+We have 3 separate sub-sets of tests, which run in parallel so it is "only" approximately 10 hours per each sub-set.
+Additionally, there is a mvn profile for the main groups - `acceptance`, `regression`, `smoke`, `bridge` and `all`,
+but we suggest to use a profile with id `all` (default) and then include or exclude specific groups.
+If you want specify the profile, Strimzi uses maven in its build and you can run particular test groups using a maven 
+profile. 
+Here is an example of how we trigger some profiles:
+
+```
+Generic definition
+
+mvn -f ${workspace}/systemtest/pom.xml -P all verify -Dgroups=${testProfile} -DexcludedGroups=${excludeGroups}
+
+Acceptance profile
+
+mvn -f ${workspace}/systemtest/pom.xml -P all verify -Dgroups=acceptance -DexcludedGroups=""
+```
+
+All available test groups are listed in the [Constants](https://github.com/strimzi/strimzi-kafka-operator/blob/master/systemtest/src/main/java/io/strimzi/systemtest/Constants.java)
+class. If you're interested, you can view our testing [document](https://github.com/strimzi/strimzi-kafka-operator/blob/master/development-docs/TESTING.md).
+
+## How system tests run in relation with Kubernetes
+
+As mentioned in the previous section, system tests run against a real Kubernetes cluster.
+You can run these tests against any Kubernetes cluster from `Minikube`, through to a big `Kubernetes` or `Openshift` cluster.
+In the `test` module we have a representation of a cluster that determines which environment it is running on.
+Moreover, we are using `Azure pipelines` to trigger these tests and using `Minikube`.
+
+## How to run system tests
+
+Running system tests is nothing to be scared of. In fact, it's easy. 
+
+The steps are as follows:
+
+1. Fork the [strimzi-kafka-operator](https://github.com/strimzi/strimzi-kafka-operator)
+2. Build the whole project in root directory - `mvn clean install -DskipTests`
+3. Set up your Kubernetes cluster
+4. Log in to your Kubernetes cluster
+5. Try to start some of our tests inside system the `test` module, for instance - `io.strimzi.systemtest.bridge.HttpBridgeST.testSendSimpleMessage()` (this takes around 5 minutes)
+
+It will run tests against the images as they are published in `Dockerhub` by default. 
+If you want to use your local images you should take a look on these environment variables:
+
+1. DOCKER_ORG = Specify the organization/repo containing the image used in system tests |`strimzi`|
+2. DOCKER_TAG = Specify the image tags used in system tests |`latest`|
+3. DOCKER_REGISTRY = Specify the docker registry used in system tests |`docker.io`|
```

Comment:
This doesn't format well ... check the preview: https://deploy-preview-187--strimzi.netlify.app/blog/2020/09/07/introduction-to-system-tests/